src/main/java/net/mcreator/element/converter/v2024_2/HarvestLevelProceduresRemover.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.converter.v2024_2;
+
+import net.mcreator.element.converter.ProcedureConverter;
+import net.mcreator.element.types.Procedure;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.Arrays;
+import java.util.List;
+
+public class HarvestLevelProceduresRemover extends ProcedureConverter {
+
+	// Starting Minecraft 1.20.6, there is no harvest level concept anymore,
+	// so there is no conversion route but to use placeholder value of 0 in those cases
+	private static final String PLACEHOLDER_BLOCK = """
+			<block type="math_number">
+				<field name="NUM">0</field>
+				<comment h="92" w="220">Harvest level system no longer exists so this block was automatically replaced with placeholder value of 0</comment>
+			</block>
+			""";
+
+	private static final List<String> HARVEST_LEVEL_BLOCKS = Arrays.asList("blockat_harvestlevel",
+			"item_get_harvest_level");
+
+	@Override public int getVersionConvertingTo() {
+		return 63;
+	}
+
+	@Override protected String fixXML(Procedure procedure, String xml) throws Exception {
+		DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+		DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+		Document doc = dBuilder.parse(new InputSource(new StringReader(xml)));
+		doc.getDocumentElement().normalize();
+
+		NodeList nodeList = doc.getElementsByTagName("block");
+		for (int i = 0; i < nodeList.getLength(); i++) {
+			Element element = (Element) nodeList.item(i);
+			if (HARVEST_LEVEL_BLOCKS.contains(element.getAttribute("type"))) {
+				DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+				DocumentBuilder builder = factory.newDocumentBuilder();

```

Comment:
Isn't it possible to use `dBuilder` declared at the beginning to not re-instantiate these 2 every iteration?

Reply:
I have copied existing converters for this, I also think builder is immutable / can't be reused without initializing it again